releaseNotes/JavaScriptFeatureManagement.md

```diff
@@ -0,0 +1,29 @@
+# Microsoft Feature Management for JavaScript
+
+[Source code][source_code] | [Package (npm)][package] | [Samples][samples]
+
+## 1.0.0 - Sep 26, 2024
+
+This is the first stable release of the following features.
+- Loading of feature flags from object and map.
+- Feature flag status evaluation with or without context.
+- Feature filters including built-in filters [`Microsoft.TimeWindow`](https://github.com/microsoft/FeatureManagement/blob/main/Schema/FeatureFilters/Microsoft.TimeWindow.v1.0.0.schema.json) and [`Microsoft.Targeting`](https://github.com/microsoft/FeatureManagement/blob/main/Schema/FeatureFilters/Microsoft.Targeting.v1.0.0.schema.json).
+
+### Breaking Changes
+
+* The feature flag provider handles duplicate flags by using the last flag instead of the first. [#40](https://github.com/microsoft/FeatureManagement-JavaScript/pull/40)
+
+### Enhancements
+
+* The targeting filter can work properly in non-Node.js environments. [#25](https://github.com/microsoft/FeatureManagement-JavaScript/pull/25)
+
+## 1.0.0-preview.1 - May 17, 2024
+
+Added support for the following features.
+- Loading of feature flags from object and map.
+- Feature flag status evaluation with or without context.
```

Comment:
Is this what you mean?

Feature flag evaluation with ambient context or user-provided context.

Reply:
No. 

My original intention is to say you can call isEnabled("MyFeature") or isEnabled("MyFeature", context) to evaluate a feature flag, depending on whether the feature flag uses any contextual feature filter.

I don't mean to call out ambient context. Because we have made the library-level decision for targeting context design that we require users to always provide targeting context. I used to think for any contextual feature filter in JS FM, if you want to use any context for the filter, you should pass it as the parameter of the isEnabled call.

I just realized that the time window filter was actually using the "ambient context" (or do we consider the current time as "ambient context"?). In .NET FM, we have IContextualFilter and IFeatureFilter. But I just releazed all feature filters need some kind of context to do the evaluation. So the context must be ambient or provided as a parameter.

So now, I think your suggestion is good. `Feature flag evaluation with ambient context or user-provided context.`